eng/pipelines/templates/jobs/stress-cluster-provision.yml

```diff
@@ -17,12 +17,17 @@ jobs:
       name: azsdk-pool-mms-ubuntu-2204-general
       vmImage: ubuntu-22.04
     steps:
-      - pwsh: |
-          $subscriptionConfiguration = @'
-            $(sub-config-azure-cloud-test-resources)
-          '@ | ConvertFrom-Json -AsHashtable;
-
-          tools/stress-cluster/cluster/provision.ps1 `
+      - task: AzureCLI@2
+        displayName: Build and Deploy Stress Tests
+        inputs:
+          ${{ if eq(parameters.Environment, 'prod') }}:
+            azureSubscription: azure-sdk-tests-public
+          ${{ if eq(parameters.Environment, 'pg') }}:
+            azureSubscription: Azure SDK Playground
```

Comment:
We should probably move all these to TME at some point.

Reply:
The playground sub? Yeah agreed, I've been starting to give people a heads up here and there.